### PR TITLE
openssl wrapper

### PR DESCRIPTION
**Pull request by sebastianbaginski**
@thomasfanell

### CODE REVIEW

**Comment by thomasfanell** [comment]
@sebastianbaginski will check it out a bit later :+1:

File: sdk/ssl/ssl.ooc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+include openssl/aes, openssl/crypto, openssl/evp, openssl/rand
+
+AesKey: cover from AES_KEY
```
Comment (marcusnaslund) [inline sdk/ssl/ssl.ooc]: I'll just say this should be `AESKey`.

Comment (sebastianbaginski) [inline sdk/ssl/ssl.ooc]: _You have no power here, Gandalf the Grey._

Comment (marcusnaslund) [inline sdk/ssl/ssl.ooc]: Apparently I do. :)

**Comment by sebastianbaginski** [comment]
I don't think this will be needed.

**Comment by thomasfanell** [comment]
Yeah sorry for the extra work :-/

**Comment by sebastianbaginski** [comment]
No problem, I used it in to generate test data in my project some time ago :D

**Pull request closed without merging**